config/secrets.yml

```diff
@@ -15,7 +15,7 @@ default: &default
   omni_contacts_google_secret: <%= ENV['OMNI_CONTACTS_GOOGLE_SECRET'] %>
   aws_access_key_id: <%= ENV['AWS_ACCESS_KEY_ID'] %>
   aws_secret_access_key: <%= ENV['AWS_SECRET_ACCESS_KEY'] %>
-  aws_bucket: <%= ENV['AWS_BUCKET'] %>
+  fog_directory: <%= ENV.fetch('FOG_DIRECTORY', "public/uploads") %>
```

Comment:
NB, important: We'll need to add an additional ENV variable, 'FOG_DIRECTORY', in clone / test / production, which is set to the current value of AWS_BUCKET.